components/bl-double-calendar-component/src/lib/classnames.js

```diff
@@ -0,0 +1,48 @@
+/*!
```

Comment:
Is it used anywhere? I could find `classnames` only in this file, and as i see it's registered and exported as file using this name.

You could try to remove it, and check if something changed.

Reply:
`classnames` is required in `react-datepicker.min.js`,  i tried removing the classnames and the component broke